shuup/core/models/_suppliers.py

```diff
@@ -30,7 +30,12 @@ class Labels:
         EXTERNAL = _('external')
 
 
-class SupplierQueryset(TranslatableQuerySet):
+class SupplierManager(TranslatableManager):
+
+    def get_queryset(self):
```

Comment:
Dos this work if I do `Supplier.objects.enabled()`?

Reply:
added
- SupplierManager tests
- SupplierDeleteView and tests